conversions/Backwards/src/main/java/com/agentdid127/resourcepack/backwards/BackwardsPackConverter.java

```diff
@@ -15,71 +15,91 @@
 import java.util.Objects;
 
 public class BackwardsPackConverter extends PackConverter {
-
     Path INPUT_DIR;
 
-    public BackwardsPackConverter(String from, String to, String light, boolean minify, Path input, boolean debug, PrintStream out) {
+    public BackwardsPackConverter(String from, String to, String light, boolean minify, Path input, boolean debug,
+            PrintStream out) {
         GsonBuilder gsonBuilder = new GsonBuilder().disableHtmlEscaping();
-        if (!minify) gsonBuilder.setPrettyPrinting();
+        if (!minify)
+            gsonBuilder.setPrettyPrinting();
         gson = gsonBuilder.create();
         DEBUG = debug;
         PackConverter.out = out;
         this.INPUT_DIR = input;
-
         log(from);
         log(to);
-
         converterRunner(from, to, light);
-        
     }
-    
+
     protected void converterRunner(String from, String to, String light) {
-        // this needs to be run first, other converters might reference new directory names
-        this.registerConverter(new NameConverter(this, Util.getVersionProtocol(gson, from), Util.getVersionProtocol(gson, to)));
+        // this needs to be run first, other converters might reference new directory
+        // names
+        this.registerConverter(
+                new NameConverter(this, Util.getVersionProtocol(gson, from), Util.getVersionProtocol(gson, to)));
```

Comment:
I prefer this on one line for stylistic reasons

Reply:
formatter